packages/frontend/src/view/components/common/icons/L2BeatIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+export function L2BeatIcon(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
I'd move `L2BeatIcon`, `L2BeatMinimalIcon`, `DydxIcon` and `StarkWareIcon` from `icons/` to `logos/` and rename to `L2BeatLogo`, `L2BeatLogoSquare`, `DydxLogoSquare` and `StarkWareLogo`.

This doesn't apply to things like `DiscordIcon` because even though this is a logo we treat it like a small monochrome icon.

Reply:
That's good catch. What do you think about creating new folder inside view called assets? Inside it I would create logos and icons

Reply:
Cuz its neither component or page